app/views/directives/edit-environment-variables.html

```diff
@@ -32,6 +34,21 @@
       add-row-with-selectors-link="Add Environment Variable Using a Config Map or Secret"
       show-header>
     </key-value-editor>
+
+    <h4>
+      Environment From
+      <span class="pficon pficon-info"
+            aria-hidden="true"
+            data-toggle="tooltip"
+            data-original-title="Environment From lets you add all key value pairs from a config map or secret."></span>
```

Comment:
"Environment From lets you add all key-value pairs from a config map or secret as environment variables."

Reply:
Copy update, np